pymc/aesaraf.py

```diff
@@ -1004,13 +970,20 @@ def compile_pymc(
         if isinstance(random_var.owner.op, RandomVariable):
             rng = random_var.owner.inputs[0]
             if not hasattr(rng, "default_update"):
-                rng_updates[rng] = random_var.owner.outputs[0]
+                update_map = {rng: random_var.owner.outputs[0]}
             else:
-                rng_updates[rng] = rng.default_update
+                update_map = {rng: rng.default_update}
```

Comment:
Nit:
```
            if hasattr(rng, "default_update"):
                update_map = {rng: rng.default_update}
            else:
                update_map = {rng: random_var.owner.outputs[0]}
```

Reply:
Or if you want to get fancy:
```
if update_val := getattr(rng, "default_update", None):
    update_map = {rng: update_val}
else:
    update_map = {rng: random_var.owner.outputs[0]}
```

Reply:
I think walrus is not in our oldest supported Python version

Reply:
🔜 then